articles/deploying-crowdstrike-with-fleet.md

```diff
@@ -0,0 +1,139 @@
+
+```
+#!/bin/bash
+CUSTOMER_ID="YOUR-CUSTOMER-ID-HERE"
+FALCON_PATH="/Applications/Falcon.app/Contents/Resources/falconctl"
+
+sudo "$FALCON_PATH" license "$CUSTOMER_ID"
+
+# Check status
+if [ $? -eq 0 ]; then
+    echo "Activation completed"
+else
+    echo "Activation failed"
+    exit 1
+fi
+```
+
+>If your org is using GitOps and you want to pass the site key as a secret, follow this guide: https://fleetdm.com/guides/secrets-in-scripts-and-configuration-profiles
+
+>For admins that are leveraging the macOS Setup Experience in Fleet, we recommend adding the software to the list of items installed on first boot.
+
+## Windows
+
+CrowdStrike offers `.exe` and `.msi` Falcon installers for Windows. Using the `.msi` is preferred as this installer type performs a silent, fully-automated installation when using the **Automatic install** option in Fleet.
+
+### Installer + script
+
+After downloading the latest CrowdStrike Falcon `.msi` installer and retrieving your Customer ID from your admin console, navigate to the **Software** tab in Fleet, then select **Add software** > **Custom package**. Upload the installer from the previous step. Select **Automatic install** or **Self-service** if those options apply to your environment. 
+
+The CrowdStrike Falcon tenant needs to collect the Customer ID from the host at time of install which we can be done with an **Install Script**. Copy and paste the code snippet below in Fleet and populate the value of the `$FalconCid` variable with your Customer ID string:
+
+```
+# Set your Customer ID here
+$FalconCid = "YOUR-CUSTOMER-ID-HERE"
+
+$logFile = "${env:TEMP}/fleet-install-software.log"
+try {
+$installProcess = Start-Process msiexec.exe `
+  -ArgumentList "/quiet /norestart /lv ${logFile} /i `"${env:INSTALLER_PATH}`" CID=${FalconCid}" `
+  -PassThru -Verb RunAs -Wait
+Get-Content $logFile -Tail 500
+Exit $installProcess.ExitCode
+} catch {
+  Write-Host "Error: $_"
+  Exit 1
+}
+```
+
+There are several other optional flags that can be added. Check the documentation for a list of supported options and their functionality.
+
+## Linux
+
+As in previous steps, find the latest installer for your Linux distro and **Download**.
+
+From the **Software** tab in Fleet, select **Add software** > **Custom package**. Upload the installer. Select **Automatic install** or **Self-service** if those options apply to your environment.
+
+### Post-install script
+
+The default install script that is populated in Fleet is sufficient, but, a post-install script is needed to set the site token and start the agent services. Below is an example post-install script that will set the token, start the service and check the status. Adjust the sleep time if needed. Also, populate the value of the `$FalconCid` variable with your Customer ID string.
```

Comment:
```suggestion
The **Customer ID** is used to assign hosts to your tenant and validate the CrowdStrike Falcon license using a script that calls the `falconctl` binary.
> Your Customer ID can be found on the Sensor download page.
In Fleet, define the following post-install script for the CrowdStrike Falcon installer, populating the value of the `$FalconCid` variable with your Customer ID string:
```

Reply:
Again, I think this instruction should be consistent with the instructions for macOS and Windows, but I'm less confident with this one since your original paragraph for Linux had a different style than the others. What do you think?